src/cpp/processor/Processor.cpp

```diff
@@ -313,12 +320,15 @@ bool Processor::process_delete_submessage(ProxyClient& client, InputPacket& inpu
 
             /* Store message. */
             output_packet.message = OutputMessagePtr(new OutputMessage(status_header));
```

Comment:
Maybe is too perfectionist, but this line is also in the `if` branch, so I think it can be factorized and put after the `if/else` once.

Reply:
Yes, it is a perfectionist, but I like it. Great!